content/06.Sampling.md

```diff
@@ -1,13 +1,28 @@
 
 ## Probability vs. Non-probability sampling 
+For a sample to correctly represent a population, several different factors (like appropriate sampling technique, sample size, parameters of interest, etc.) need to be considered. So, the next question here is how to draw a sample. There are mainly two types of sampling designs: probability and non-probability sampling. These designs differ in terms of reliability and method of parameter estimation.
+
+Probability sampling: In the probability sampling scheme, every unit from the sample frame has a non-zero probability of selection. It, therefore, leads to unbiased estimates of the mean and variance for the variable of interest. Therefore, any method aimed at generalizing results drawn by a sample to the whole population of interest must be based on probability sampling. 
+
+Non-probability sampling: In non-probability sampling, researchers select samples based on their own convenience. In other words, the researchers purposively choose particular individuals/units/elements for constituting a sample. Social researchers often use this sampling design to select households or families to conduct their surveys. A few common techniques are convenience sampling, judgment sampling, quota sampling, and snowball sampling.
 
 ## Defining the population, sampling frame, sampling unit, etc. 
+Population (target population): Collection of all individuals of one species inhabiting an area at a given time, about which some parameters of interest are to be estimated. In wildlife studies, this could be all the individuals of a species or subspecies in a habitat. 
+
+Sample unit: A unique collection of elements (e.g., plots or organisms) on which sample data are collected.
+
+Sampling frame: A finite set of all individuals that can be measured. It usually coincides with the target population, but reasons like accessibility can make it differ otherwise.
 
 ## Simple random sampling 
+In simple random sampling, every unit has an equal probability of inclusion in the sample.
 
 ## Stratified random sampling 
+Stratified sampling is generally used when the population from which the sample is to be drawn does not belong to one homogeneous group and there is a high variation within the population. This sampling technique then divides the whole population into different groups (strata) according to some characteristics such as the habitat they inhabit, gender, etc. Ideally, the strata should be homogeneous with respect to the variable of interest (like density, abundance, etc.). Once we have these different strata, a random sample is then drawn from each stratum based on the percentage that each subgroup represents in the population. The stratified sampling method is common in wildlife studies, as it helps in estimating parameters within strata and contrasting parameters among strata. 
 
-## Systematic sampling 
```

Comment:
You should consider adding other common sampling approaches too like adaptive sampling, cluster sampling, for example. 

You should also include a box that shows examples of these different methods with associated code.

Reply:
Sure. I will do that.